app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -276,13 +277,13 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
     private var emojiPickerVisible = false
 
     private val isScrolledToBottom: Boolean
-        get() = binding?.conversationRecyclerView?.isScrolledToBottom ?: true
+        get() = binding.conversationRecyclerView?.isScrolledToBottom ?: true
```

Comment:
don't think you need the `?` after the recyclerview ( which means you don't need the ?: true either )

Reply:
All redundant null checks removed.